kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -330,71 +363,48 @@ define([
                     }
                 }],
                 rowCallback: function (row) {
-                    var getFileFromName = function (fileName) {
+                    const getFileFromName = function (fileName) {
                         return files.filter(function (file) {
                             return file.name === fileName;
                         })[0];
                     };
 
-                    $('td:eq(2)', row).find('.kb-staging-table-body__name').tooltip({
```

Comment:
This chunk was moved down below, I reordered everything so we're declaring these in order (0-5)